packages/pf4-component-mapper/src/select/select/option.js

```diff
@@ -19,6 +21,7 @@ const Option = ({ item, isActive, isSelected, ...props }) => (
           <CheckIcon />
         </span>
       )}
+      {item.description && <div className="ddorg__pf4-component-mapper__select-option-description">{item.description}</div>}
```

Comment:
@rvsia is it possible to use the PF class for this? As we do with the rest? I think we will hear about this not being a PF component if we don't use them.

Reply:
Yeah, that was the first thing I tried to do. However, we do not import the dropdown component so the PF class styles are missing. (And the whole select is custom styled anyways..)

Reply:
*the class was there, but the dropdown specific variables were missing.

Reply:
Yup, we should use the class. And you can always import the CSS from @patternfly/react-styles to inject them.